scanamo/src/main/scala/org/scanamo/ScanamoFree.scala

```diff
@@ -70,7 +71,7 @@ object ScanamoFree {
           batch,
           item =>
             WriteRequest.builder
-              .putRequest(PutRequest.builder.item(f.write(item).asObject.getOrElse(DynamoObject.empty).toJavaMap).build)
+              .putRequest(PutRequest.builder.item(f.write(item).asObject.orEmpty.toJavaMap).build)
```

Comment:
Sneaking this unrelated change into this PR - should have done it when we introduced `Monoid[DynamoObject]` in #1796.